src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
-body {
+@import "variables.scss";
+@import "card.scss";
+@import "stars.scss";
+
+body,
+img,
+div,
+a,
+p {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Helvetica, sans-serif;
 }
```

Comment:
All of those 3 properties usually are not written together
1. Remember, that `box-sizing: border-box` we define 99% of the time as 
```
* {
  box-sizing: border-box;
}
```
2. We reset margins only for those elements that really have them by default
3. Global font related styles are better defined in `html` or `:root` elements, so....
```
* {
  box-sizing: border-box;
}

:root {
  font-family: Roboto, Helvetica, sans-serif;
}

body,
p,
h2 {
  margin: 0;
}
```

⚠️ Pay attention that I added `<h2>`, cause it has default margins.